src/components/exhibition-detail-page/exhibition-detail-page.stories.js

```diff
@@ -0,0 +1,15 @@
+import html from "../../../.storybook/helpers/html";
```

Comment:
I think this file getting included might be an accident. Should we delete?

Reply:
I mean this entire file (src/components/exhibition-detail-page/exhibition-detail-page.stories.js) that is

Reply:
Yeah, good call